climpred/prediction.py

```diff
@@ -260,6 +273,12 @@ def compute_hindcast(
                     )
                 }
             )
+        # If dask, then chunk in time.
```

Comment:
Is this really needed? Chunking might be more effective on non climpred dimension. I would prefer not to have these lines? Rather having a check than warns if chunking is not useful in that way

Reply:
What you actually want is that time is not chunked into many pieces, right? Then rather check for this. Or does the current implementation allow still for spatial chunks?

Reply:
Yeah this is needed. It breaks if it's not there. Since operations are being applied over the entire time dimension at once (e.g. correlation), it needs to be one chunk. This just makes sure that time is chunked into one chunk and leaves the other dimensions as they were.

Reply:
I think we should just prevent more than one chunk in the time dimension. I think the current implementation prevents spatial chunks. Unsure whether this is true. 

But I can address this in a later PR. This is minor.

Reply:
Can you just check with a print(a.chunks) before and after this re-chunking?

Reply:
Yeah I will give this a look.

Reply:
Forget about it. I this the current code is absolutely fine.